src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Utilities/ICollectionExtensions.cs

```diff
@@ -29,12 +29,16 @@ public static void AddRange<T>(this ICollection<T> collection, IEnumerable<T>? v
         public static void AddRange<T>(this ICollection<T> collection, HashSet<T>? values)
         {
             if (collection == null)
+            {
                 throw new ArgumentNullException(nameof(collection));
+            }
 
             if (values != null)
             {
                 foreach (var item in values)
+                {
                     collection.Add(item);
+                }
             }
         }
```

Comment:
revert.

Reply:
These were pulled from the original commit to match the other AddRange methods. Removed them again.